tools/query_breakdown/src/main/java/com/google/bigquery/LocationTracker.java

```diff
@@ -95,5 +95,4 @@ public LocationTracker cloneTracker() {
     return locationTracker;
   }
 
-
```

Comment:
Again, pls remove this file from the pull request.

Reply:
removed!